packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -93,7 +93,7 @@ const AdvertiserPage = () => {
                     page_title={localize("Advertiser's page")}
                 />
                 <MobileWrapper>
-                    <AdvertiserPageDropdownMenu is_my_advert={is_my_advert} />
+                    {!is_my_advert && <AdvertiserPageDropdownMenu is_my_advert={is_my_advert} />}
```

Comment:
since we are not showing the dropdown on the advertiser's page, can we remove `is_my_advert` prop from `AdvertiserPageDropdownMenu` since it's no longer needed?